src/components/PeopleTable.tsx

```diff
@@ -1,682 +1,171 @@
+                      : null,
+                  }}
+                >
+                  <span className="icon">
+                    <i
+                      className={cn(
+                        'fas',
+                        'fa-sort',
+                        {
+                          'fa-sort-up':
+                            sort === tableName.toLowerCase()
+                              && !order,
+                        },
+                        {
+                          'fa-sort-down':
+                            sort === tableName.toLowerCase()
+                              && order === 'desc',
+                        },
+                      )}
+                    />
+                  </span>
+                </SearchLink>
+              </span>
+            </th>
+          ))}
+          <th>Mother</th>
+          <th>Father</th>
         </tr>
+      </thead>
 
-        <tr data-cy="person">
-          <td>
-            <a href="#/people/emile-haverbeke-1877">Emile Haverbeke</a>
-          </td>
-          <td>m</td>
-          <td>1877</td>
-          <td>1968</td>
-          <td>
-            <a
-              className="has-text-danger"
-              href="#/people/maria-sturm-1835"
-            >
-              Maria Sturm
-            </a>
-          </td>
-          <td>
-            <a href="#/people/carolus-haverbeke-1832">Carolus Haverbeke</a>
-          </td>
-        </tr>
+      <tbody>
+        {sortedPeople().map(({
+          name,
+          sex,
+          born,
+          died,
+          fatherName,
+          motherName,
+          slug: link,
+        }) => (
+          <tr
+            key={name}
+            data-cy="person"
+            className={cn(
+              { 'has-background-warning': slug === link },
+            )}
+          >
+            <td>
+              <Link
+                to={`/people/${link}`}
+                className={cn(
+                  { 'has-text-danger': sex === 'f' },
+                )}
+              >
+                {name}
+              </Link>
+            </td>
+
+            <td>{sex}</td>
+            <td>{born}</td>
+            <td>{died}</td>
+            {personLink(motherName) ? (
+              <td>
+                <Link
+                  to={`/people/${personLink(motherName)}`}
+                  className="has-text-danger"
+                >
+                  {motherName || '-'}
+                </Link>
+              </td>
+            ) : (
+              <td>
+                {motherName || (
+                  '-'
+                )}
+              </td>
+            )}
+            {personLink(fatherName) ? (
+              <td>
+                <Link
+                  to={`/people/${personLink(fatherName)}`}
+                >
+                  {fatherName || '-'}
+                </Link>
+              </td>
+            ) : (
+              <td>
+                {fatherName || (
+                  '-'
+                )}
```

Comment:
```suggestion
                {fatherName ||  '-'}
```

no need to write it from a new line